x/distribution/simulation/operations_test.go

```diff
@@ -244,7 +246,7 @@ func (suite *SimTestSuite) getTestingValidator(accounts []simtypes.Account, comm
 	valPubKey := account.PubKey
 	valAddr := sdk.ValAddress(account.PubKey.Address().Bytes())
 	validator, err := stakingtypes.NewValidator(valAddr, valPubKey, stakingtypes.
-		Description{})
+	Description{})
```

Comment:
```go
validator, err := stakingtypes.NewValidator(valAddr, valPubKey, stakingtypes.Description{})
```
How about removing the newline?

Reply:
This is not my changes, but it seems strange. ^^;
I'll fix it but, please approve again.